### ISSUE TITLE
BUSTED results interpretation

### ISSUE THREAD

**Issue by sadiexiaoyu** [veg/hyphy#379]
Dear All, 
I am running BUSTED now. Could anyone give a clue how to interpret the results? For example, what are the Constrained Evidence Ratio, Optimized Null Evidence Ratio, Optimized Null Likelihood mean? And is BUSTED only give the sites which are under positive selection? Can I also see which branch is under positive selection? Or is there any manual can give a detail explanation? I always find that Hyphy manual is not as easy as PAML to understand...

**Comment by spond** [comment]
Dear @sadiexiaoyu,

Please take a look at these documents:

0. You can go back to the original papers (see item 3 for links); we try to write them in a way that is fairly straight-forward. 

1. [A relatively simple overview](http://bit.ly/pond-hutch-2015) of how we think selection analyses should be done in HyPhy (a recent talk I gave) 

2. See a [simple tutorial](https://github.com/veg/hyphy-tutorials/tree/master/selection#run-alignment-wide-tests-for-episodic-diversification-busted) for how BUSTED should be run/interpreted. 

3. You **can** use BUSTED to both test for selection along some (pre-specified) branches, and screen individual sites for selection, but it is not **best** at these tests. It is **designed** to answer the question about whether or not the entire gene is under selection (somewhere in time, and and somewhere along the sequence, i.e. in the branch-site framework)

Best,
Sergei

**Comment by sadiexiaoyu** [comment]
Dear Sergei,
Thanks for the reply! So the BUSTED most focus on whether part of the sites are under selection, similar as branch-site model in PAML, but BUSTED can be also done without priori specified background, right? Then the aBSREL should be the better method to show me whether specific lineage is under selection, if I understand correctly? 

I am confusing about the data results, now. First of all, I used GARD to test whether there are recombination in the alignment. Then I run branch-site model in PAML which give me very good results actually fit to my hypothesis very well. However, when I run aBSREL in http://test.datamonkey.org/absrel, the result shows that 0 is under selection. Actually, I test two cases: one is with duplicated gene, and the other one is only with homologous gene. However, both of the cases shows that PAML gave very good significant results, while aBSREL don,t... I really don,t know why... The results should be not so different, right? Are these two methods have different requirment for the testing sequences? I found that people said Hyphy mostly focus on testing Virus, but PAML for other organisms... Is this true?

Sadiexiaoyu
